test/Test/intlinear.jl

```diff
@@ -24,9 +24,9 @@
     MOIU.set_mock_optimize!(mock,
         (mock::MOIU.MockOptimizer) -> MOIU.mock_optimize!(mock, [1.25, 8.75, 0., 1.])
     )
-    MOIT.indicator_test1(mock, config)
+    # MOIT.indicator1_test(mock, config)
```

Comment:
Why is it commented ?

Reply:
`MOIT.indicator1_test(mock, config)`
The `mock` optimizer does not support indicator constraints, I found nowhere to change this. Furthermore, `SOS` constraints are not tested here either

Reply:
It should be added here:
https://github.com/JuliaOpt/MathOptInterface.jl/blob/589cd2252c5fb1aacbfb0852a931a8a703265746/test/runtests.jl#L28-L29
You should do something like
```julia
const LessThanIndicatorSet{T} = IndicatorSet{MOI.ACTIVATE_ON_ONE, MOI.LessThan{T}}
```
and add `LessThanIndicatorSet` in the macro because in this macro, the sets have either no type argument or only `T`, the coefficient type.